src/components/Dropdown/DropdownTrigger.react.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+import * as React from "react";
+import cn from "classnames";
+import { Icon, Button } from "../";
+
+type Props = {|
+  +children?: React.Node,
+  +className?: string,
+  +toggle?: boolean,
+  +value?: string,
+|};
+
+function DropdownTrigger({
+  className,
+  toggle,
+  value,
+  children,
+  ...rest
+}: Props): React.Node {
+  const classes = cn({ "dropdown-toggle": toggle }, className);
+  return (
+    <Button className={classes} data-toggle="dropdown" {...rest}>
+      {value && value}
```

Comment:
Are you able to just do `{value}`?  I think if it is null then React will ignore it but I could be wrong.  Same with line 24.